packages/gitbook/src/components/primitives/Button.tsx

```diff
@@ -15,7 +15,6 @@ type ButtonProps = {
 
 export function Button({
     href,
-    onClick,
```

Comment:
Can you add an E2E test for cookie banner accept?

Reply:
I can, but it'll slow down the "hotfix".